### ISSUE TITLE
Decide where to host Kata 2.x releases

### ISSUE THREAD

**Issue by jodh-intel** [kata-containers/kata-containers#745]
Currently, we post releases for Kata 1.x in the Kata 1.x runtime repo at https://github.com/kata-containers/runtime/releases.

However, the Kata 2.x code lives in a different repository (this one!) so it makes sense in my mind to host the Kata 2.x releases at https://github.com/kata-containers/kata-containers/releases since:

- It will be confusing to put releases in the Kata 1.x repo when the development changes are happening in a different repo.
- We don't have a documented upgrade path from 1.x to 2.x so it's "safest" to keep the release streams separate currently.
- Eventually, https://github.com/kata-containers/runtime/releases will be archived as the code in it is fully replaced by the runtime code in this repo and all maintained stable releases are part of the 2.x development stream.

# Questions

- Should we create a new repo just for releases (https://github.com/kata-containers/releases)?
- Could we instead simply move all new releases (1.x and 2.x) to https://github.com/kata-containers/kata-containers/releases?

In summary, we need a stable and reliable URL for both users and tooling (such as added by https://github.com/kata-containers/kata-containers/pull/735) to know where to find release details.

wdyt @gnawux, @amshinde, @jcvenegas?

**Comment by jodh-intel** [comment]
Also affects https://github.com/kata-containers/kata-containers/pull/739.

**Comment by bergwolf** [comment]
All the 2.0 alpha releases are hosted in the kata-containers repository. E.g., https://github.com/kata-containers/kata-containers/releases/tag/2.0.0-alpha3

**Comment by jodh-intel** [comment]
Thanks @bergwolf - closing! ptal at https://github.com/kata-containers/kata-containers/pull/735 btw.
